codec-http/src/main/java/io/netty5/handler/codec/http/websocketx/WebSocketClientHandshaker13.java

```diff
@@ -267,12 +232,12 @@ protected FullHttpRequest newHandshakeRequest(BufferAllocator allocator) {
      */
     @Override
     protected void verify(FullHttpResponse response) {
-        HttpResponseStatus status = response.status();
+        var status = response.status();
```

Comment:
Not sure what the consensus is on using the `var` keyword.

Reply:
Here is a good guide - https://res.cloudinary.com/snyk/image/upload/v1524742320/Local_Type_Inference_Cheat_Sheet_for_Java_10_and_beyond_web.png

Reply:
@doom369  thanks, good guide and It's looks similar with my list, also i never use var for base types (int, byte, .....). By the way, again if you think not use var at all in our code i can remove, no problem here. Or we can approve some cases in its use and follow them otherwise as I mentioned there are no strict rules and each contributor will do it in his own way. @doom369, @hyperxpro, @chrisvest, @normanmaurer WDYT ?

Reply:
I'd be fine with following the guidelines of that cheat sheet.

Reply:
> also i never use var for base types (int, byte, .....).

Same here, also I never do that for the method returns, like in this particular example, as it always complicates the code reading, even the variable name seems super straightforward.

Reply:
Fixed, according guidelines.